packages/shared/src/lib/core/layer-2/actions/getLayer2NetworkFromAddress.ts

```diff
@@ -1,11 +1,7 @@
 import { getActiveProfile } from '@core/profile/stores'
-import { EvmNetworkType, IIscpEvmNetworkConfiguration } from '@core/network'
 
 export function getLayer2NetworkFromAddress(address: string): string | undefined {
     const chainConfigurations = getActiveProfile()?.network?.chainConfigurations
-    const iscpNetworks = chainConfigurations?.filter(
-        (evmNetwork) => evmNetwork.type === EvmNetworkType.Iscp
-    ) as IIscpEvmNetworkConfiguration[]
-    const network = iscpNetworks?.find((evmNetwork) => evmNetwork.aliasAddress === address)
+    const network = chainConfigurations?.find((evmNetwork) => evmNetwork.aliasAddress === address)
```

Comment:
maybe call `evmNetwork` here `l2Network` because this will only be used for that?

Reply:
I deconstructed the property directly.